doc/wiki/5_Controllers.md

```diff
@@ -1,58 +1,116 @@
+### 3.2. `fri_configuration_controller`
 
 The `SendPeriodMilliSec` parameter of FRI defines the period with which the controller sends state updates, while the `ReceiveMultiplier` defines the answer rate factor (ratio of receiving states and sending commands). These are parameters of the hardware interface, which can be modified in connected state, when control is not active. To support changing these parameters after startup, the `FRIConfigurationController` advertises the topic `~/set_fri_config`. Sending a message containing the desired integer values of `send_period_ms` and `receive_multiplier` updates the parameters of the hardware interface.
 
-__Required parameters__: None
+__Required Parameters__: None
+
+## 4. Hybrid Controllers
+
+Hybrid Controllers group together related functionalities that are intended to be used in combination. Currently, there is a single Hybrid Controller that handles non-real-time messages. It effectively combines the roles of a Broadcaster and a Configuration Controller.
+
+### 4.1. `kuka_nrt_message_handler`
+
+The `kuka_nrt_message_handler` controller provides several non-real-time capabilities:
+
+- __Toggle Robot Drives__
+
+  Publish a `std_msgs::msg::Bool` message to `~/drive_state` to control the drive state:
+
+  ```shell
+  ros2 topic pub /nrt_message_handler/drive_state std_msgs/msg/Bool "{data: false}" --once
+  ```
+
+- __Set RSI Cycle Time__
+
+  Publish a `std_msgs::msg::UInt8` message to `~/cycle_time` to set the cycle time:
+  - 1 &rarr; 4 ms
+  - 2 &rarr; 12 ms
+
+  ```shell
+  ros2 topic pub /nrt_message_handler/cycle_time std_msgs/msg/UInt8 "{data: 1}" --once
+  ```
+
+- __Monitor Robot Status__
+
+  Subscribe to `~/status` to receive updates via `kuka_driver_interfaces::msg::KssStatus`, which includes:
+
+  Field             | Possible values                          | Description
+  ------------------|------------------------------------------|--------------------------------------------------------------------
+  `control_mode`    | `1` (joint position)                     | Specifies the robot's control mode
+  `cycle_time`      | `1` (4 ms), `2` (12 ms)                  | Defines the selected RSI communication cycle time
+  `drives_powered`  | `true`/`false`                           | Shows whether the robot's drives are powered
+  `emergency_stop`  | `true`/`false`                           | Reflects whether the emergency stop is active
+  `guard_stop`      | `true`/`false`                           | Indicates whether the guard stop is active
+  `in_motion`       | `true`/`false`                           | Shows whether the robot is moving
+  `motion_possible` | `true`/`false`                           | Indicates whether motion is possible
+  `operation_mode`  | `1` (T1), `2` (T2), `3` (AUT), `4` (EXT) | Represent the robot's current operation mode
+  `robot_stopped`   | `true`/`false`                           | Signals whether there is a message that required the robot to stop
```

Comment:
Is there necessarily a message, doesn't it just simply mark that the robot is stopped?